stdlib/@python2/typing_extensions.pyi

```diff
@@ -33,7 +34,7 @@ def runtime_checkable(cls: _TC) -> _TC: ...
 
 # This alias for above is kept here for backwards compatibility.
 runtime = runtime_checkable
-Protocol: _SpecialForm = ...
+Protocol: _SpecialForm
```

Comment:
Can we re-export Protocol from typing?

Reply:
It's not over till mypy_primer sings, but looks like we can!